core/test/data_proxy_test.cpp

```diff
@@ -5,11 +5,10 @@
 
 #include <numeric>
 
+#include "scipp/common/fix_typed_test_suite_warnings.h"
```

Comment:
Just include this in `test_macros.h`, so we do not have to put it everywhere? It will just be forgotten otherwise when new files are added, and the warnings will be back.